cf-macro-serverless/serverless.yml

```diff
@@ -0,0 +1,30 @@
+service: cf-macro
```

Comment:
Since this is intended for SAM support, I'd like to see if we can go back to deploying this macro with SAM. I believe that the issue was with the dependencies on `slic-watch-core`. We can revisit this with @lmammino and see if we can find a good way to do all of this.

Reply:
Sure, removed serverless version.